<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Usuarios</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
      QMainWindow{
        background-color: #F0A500;
      }
     </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap>roles.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_insertar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QPushButton {
        background-color: #4CAF50;
        color: black;
        border: none;
        border-radius: 5px;
        padding: 10px;
      }
      QPushButton:hover {
        background-color: #45a049;
      }
     </string>
    </property>
    <property name="text">
     <string>Insertar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_eliminar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QPushButton {
        background-color: #f44336;
        color: black;
        border: none;
        border-radius: 5px;
        padding: 10px;
      }
      QPushButton:hover {
        background-color: #e53935;
      }
     </string>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_actualizar">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QPushButton {
        background-color: #2196F3;
        color: black;
        border: none;
        border-radius: 5px;
        padding: 10px;
      }
      QPushButton:hover {
        background-color: #1e88e5;
      }
     </string>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_listar">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QPushButton {
        background-color: #FFC107;
        color: black;
        border: none;
        border-radius: 5px;
        padding: 10px;
      }
      QPushButton:hover {
        background-color: #ffb300;
      }
     </string>
    </property>
    <property name="text">
     <string>Lista</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>20</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>GESTION DE USUARIOS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
